docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -385,8 +385,8 @@ To write with custom column names, use the `CSVDataNode.write_with_column_names(
 When reading from an Excel data node, Taipy returns the data of the Excel file based on the _exposed_type_ parameter.
 Check out [Excel Data Node configuration](../config/data-node-config.md#excel) for more details on _exposed_type_.
 
-For the example in this section, assume that `sales_history_cfg` in [`my_config.py`](https://github.com/Avaiga/taipy-doc/blob/develop/docs/manuals/core/my_config.py)
-is an _Excel_ data node configuration with `default_path="path/sales.xlsx"`.
+For the example in this section, assume that `sales_history_cfg` is an _Excel_ data node
```

Comment:
```suggestion
For the example in this section, assume that *sales_history_cfg* is an *Excel* data node
```
- Avoid underscores for italics so that Python identifiers are not a problem.
- References to variables in code should be italics as well - as everywhere - and no 'code', which is reserved to for the API